Modelica/Fluid/Examples/DrumBoiler.mo

```diff
@@ -70,9 +70,9 @@ package DrumBoiler
     Modelica.Blocks.Interfaces.RealOutput V_l(unit="m3") "Liquid volume inside drum"
       annotation (Placement(transformation(extent={{100,74},{112,86}})));
   public
-    Modelica.Blocks.Math.Gain MW2W(k=1e6)
+    Modelica.Blocks.Math.Gain MW2W(final k=1e6)
       annotation (Placement(transformation(extent={{-54,-75.5},{-44,-64.5}})));
-    Modelica.Blocks.Math.Gain Pa2bar(k=1e-5) annotation (Placement(
+    Modelica.Blocks.Math.Gain Pa2bar(final k=1e-5) annotation (Placement(
```

Comment:
I find it rather obscure to leave it for unit inference to figure out the unit of the gain, and I find setting `displayUnit` a good way to make clear that the numeric value is the correct one:
```suggestion
    Modelica.Blocks.Math.Gain Pa2bar(final k(unit = "bar/Pa", displayUnit = "1") = 1e-5) annotation (Placement(
```

Reply:
(If I am not mistaken about `final`, there is no point putting `final` before `unit` when the value modification is already made `final`.)

Reply:
Why not to use `Modelica.Blocks.Math.UnitConversions.To_bar` instead of gain?
Similar for `K2degC` instance which shall be `Modelica.Blocks.Math.UnitConversions.To_degC` IMO.

Reply:
I agree that ideally that should have been done, but:
- I wanted minimal changes in the model
- It should be an example of how to build similar models, and in general there might not exist such blocks for the unit you want to use; so showing that you can just use a gain-block seems like a good idea.